### ISSUE TITLE
Clearing Operations project template not packaging all PY resources - may need to combine in PYT

### ISSUE THREAD

**Issue by BobBooth** [Esri/solutions-geoprocessing-toolbox#599]
I am attempting to package the Clearing Operations ArcGIS Pro project as a Pro Project Template.
The ClearingOperationsTools.pyt packages, and I can attache the LYR/LYRX files as attachments to the project template, but the supporting modular PY files:
__init__.py
GRGTools.py
GRGUtilities.py
NumberFeaturesTool.py
Utilities.py
do not get packaged up. 
I did a proof of concept merge of the PY files into the PYT, and they do work that way (though it is a bit rough, I'm seeing the symbology not be honored, so needs some work).

**Comment by BobBooth** [comment]
@mfunk - see PYT file in the Maintenance chat.

**Comment by BobBooth** [comment]
The Clearing Operations APTX on the staging server is updated, using the merged PYT file.

**Comment by mfunk** [comment]
@BobBooth do NOT add the combined PYT to repo. Were you able to discuss with Pro/Python team?

**Comment by BobBooth** [comment]
@mfunk - I have not. Still trying to get in touch with someone on the Pro/Py team. DF is OOO. Have pinged Ghis to ask who he thinks would be next best to talk to.

**Comment by mfunk** [comment]
@BobBooth 
Can you summarize your finding with the Pro/Python/GP/etc. Teams here?

**Comment by csmoore** [comment]
Bob was never able to get a Pro .pyt toolbox that included other files (the ones in the script folder) to package in a template with the other dependent python files (the ones in the \scripts folder). 

But this is a Pro template packaging issue (that we were able to work around) so the issue in the repo can probably be closed.

**Comment by BobBooth** [comment]
@mfunk  - The Pro team said they had fixed a bug with the 2.1 release that _might_ help, and suggested trying repackaging using the GP "Package Project" tool in 2.1.  
"...Also, when creating your template package, make sure that you use the Project Package GP tool.  When using the tool, make sure  that the following parameters are checked:
-	Package as Template
-	Include Toolboxes
..."
This image was attached:
![image](https://user-images.githubusercontent.com/3801392/30866893-7c589f0c-a2a8-11e7-8faa-291777c811e6.png)
 I have not done so yet. 
No other feedback received.

**Comment by csmoore** [comment]
No change when testing with Pro 2.1 Beta (Tested on 2.1.9865).

Scripts folder still missing.

Important Note: I *believe* there is also a (little known) requirement to have run the GP Tools at least once - however I was unable to run the tools because of issue: https://github.com/ArcGIS/solutions-defense/issues/443

![image](https://user-images.githubusercontent.com/3090809/31126706-8a9e8fa0-a81a-11e7-9729-7f923f4d0cfd.png)

**Comment by topowright** [comment]
I am closing this issue and moving the traffic to the issue in Solutions Defense. 

https://github.com/ArcGIS/solutions-defense/issues/429
